src/recipe.cpp

```diff
@@ -418,22 +418,18 @@ bool recipe::has_byproducts() const
     return !byproducts.empty();
 }
 
-std::string recipe::required_skills_string( const Character *c, bool print_skill_level ) const
+// Format a std::pair<skill_id, int> for the crafting menu.
+// skill colored green (or yellow if beyond characters skill)
+// optionally with the skill level (player / difficulty)
+template<typename _FIter>
```

Comment:
Identifiers starting with an underscore followed by a capital letter are reserved for the standard library.  It can just be e.g. `Iter`, rather than `_FIter`.

Reply:
Note that I based this off existing code:

https://github.com/CleverRaven/Cataclysm-DDA/blob/master/src/output.h#L690